src/Umbraco.Web/Editors/AuthenticationController.cs

```diff
@@ -60,9 +60,9 @@ public AuthenticationController(IUserPasswordConfiguration passwordConfiguration
         /// </summary>
         /// <returns></returns>
         [WebApi.UmbracoAuthorize(requireApproval: false)]
-        public IDictionary<string, object> GetPasswordConfig()
+        public IDictionary<string, object> GetPasswordConfig(int userId)
         {
-            return _passwordConfiguration.GetConfiguration();
+            return _passwordConfiguration.GetConfiguration(userId != UmbracoContext.Security.CurrentUser.Id);
```

Comment:
I'm not a fan of this logic in a controller. It is business logic that we old allow `allowManuallyChangingPassword` for the current user..

Reply:
Yep i agree, I forgot we can access the current user on a thread without any http stuff so i can change that.

Reply:
I will leave this as-is for now, we'll be changing more later with passwords, auth, members, users and i'm positive we'll be re visiting this.